### PR TITLE
Scheinman jan webcast dec2015

### PR DESCRIPTION
**Pull request by carolineHecht**
Changes to Scheinman webcast form, also necessitated changes to their Join Mailing List form.

### CODE REVIEW

**Comment by a-fro** [comment]
@carolineHecht as we've discussed, removing field instances and bases from features doesn't remove them from the database. You will need to add an update hook to delete the fields so that they don't cause issues in the future.

**Comment by carolineHecht** [comment]
@a-fro I can add the update hook to delete those fields unless you already did.

**Comment by carolineHecht** [comment]
@a-fro Added the update hook

File: docroot/sites/all/modules/features/forms2/forms2.install [hunk 1]
```diff
@@ -0,0 +1,11 @@
+**
+ * Removes field_yes_no from scheinman_webcast_registration form and two other
+ * fields altogether
+ */
+function ilr_forms2_update_7001() {
```
Comment (a-fro) [inline docroot/sites/all/modules/features/forms2/forms2.install]: Hooks in Drupal are structured as [module_name]_[hook_name]_[other_info]. 

So, for hook_update_n (see https://api.drupal.org/api/drupal/modules%21system%21system.api.php/function/hook_update_N/7), the module name here is "forms2", not "ilr_forms2". 

Also, it's not clear to me at this point why Ted was running both field_delete_field and field_delete_instance, as the former calls the latter. Please redo this commit and force push. Thanks.

**Comment by a-fro** [comment]
@carolineHecht there are still issues with the install file. Sorry that I didn't notice before, but I was just looking at the code in github and it wasn't obvious until l pulled it down. .install files in Drupal are php files, so you need to open the php tag at the top of the file for it to work correctly. If you had done that, you also would have seen that there was a php syntax error in that file. 

Please `git reset head~2`, make the changes to the file and add it as a single commit, rather than 3. You will need to force push at that point.

**Pull request merged**